src/plugins/template/src/template_compiled_model.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
do we need `template_` at the beginning of each file name?
I suppose it's our legacy notation of naming

Reply:
Agree